### ISSUE TITLE
Server Population Issue

### ISSUE THREAD

**Issue by Awfulwaffle1** [itsamewayu/OLFallenDoll#883]
**Is your feature request related to a problem? Please describe.**
There is a bit of a mix crowd of for Harem use. Some like playing with others, others merely want to use harem or want an offline version of it, but opts to use the DND mode. 
Also, with the current server structure, players seem to only join lobbies within the closest proximity of locale (or whatever VPN they are on). Previously, players were able to select with region they'd like to join, which fixed that issue. 

**Describe the solution you'd like**
The most ideal solution:
- Being able to create lobbies/join lobbies as needed, with the option of password locking or setting it to invite only. 
This satisfies both crowds. 
Public lobbies that will enjoy playing and like-swapping.
Private lobbies for those that want to just enjoy the experiments a la gallery mode (RIP).

**Describe alternatives you've considered**
Other alternatives:
- Bring back server selection so players can shift to where the traffic is mostly.
- Set the servers populate traffic regardless of locale. (least ideal, but probably the easiest to implement temporarily)

**Comment by NGC603** [comment]
To add to this, something needs to be done to de-incentivize the use of bondage holes **solely** as a method of AFK farming, like increasing the rate of sanity drain or giving more rewards for logged-in interactions. Even with a public/private lobby solution or server selection, the issue would remain since passive reward gain is still too good of an incentive. Creating a new lobby or selecting a new server when all the holes are filled with logged-out users doesn't help much if the new lobby just gets filled with logged-out users again.

**Comment by kmoonwalker** [comment]
yeah I would like to have a selection to get a night time when few people or no one in eu is around to some company and be able to send likes etc ;)
